scripts/localdev.py

```diff
@@ -731,14 +734,19 @@ def stop_testnet(config, testnet_name):
     # tediously, testnet.sh has a habit of returning non-zero error codes when eg. the testnet has already been destroyed :-(
     run_or_die(config, ["sh", "-c", "echo localdev | ./testnet.sh down"], in_dir=os.path.join(TESTNET_DIR, testnet_name), allow_failure = True)
 
-def write_testnet_configuration(config, zilliqa_image, testnet_name, isolated_server_accounts, persistence, key_file, chain_id,desk):
+def write_testnet_configuration(config, zilliqa_image, testnet_name, isolated_server_accounts, bucket_name, persistence, key_file, chain_id, desk):
     instance_dir = os.path.join(TESTNET_DIR, testnet_name)
     minikube_ip = get_minikube_ip(config)
 
     if os.path.exists(instance_dir):
         print(f"Removing old testnet configuration ..")
         shutil.rmtree(instance_dir)
     print(f"Generating testnet configuration .. ")
+
+    # IMPORTANT: if you change any of the flags make sure to update the count returned
+    #            by this function so that prometheus_up knows how many pods to wait for.
+    # 20 normal pods (i.e. 5 ds + 15 non-ds), 1 lookup, 2 seedpubs, 1 seedprv.
+    count = 20 + 1 + 2 + 1
```

Comment:
This seems dodgy - what happens if this count isn't right? Will the user get a reasonable error message, or will they have to go hunting for why everything's bust .. ?

Reply:
Accidental.